lib/iarc_v2/client.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf8 -*-
+import datetime
+import json
+import requests
+from urlparse import urljoin
+
+import commonware.log
+from django.conf import settings
+
+from lib.iarc_v2.serializers import IARCV2RatingListSerializer
+
+
+log = commonware.log.getLogger('z.iarc_v2')
+
+
+def get_rating_changes():
+    """
+    GetRatingChange for all changes within the last day.
+
+    TODO: Could add support for pagination, but very low priority since we
+          will never ever get anywhere close to 500 rating changes in a single
+          day.
+    """
+    if settings.IARC_V2_MOCK:
+        return json.loads(file('./mock/GetRatingChanges.json').read())
+    else:
+        start_date = datetime.datetime.utcnow()
+        end_date = start_date - datetime.timedelta(days=1)
+        url = urljoin(settings.IARC_V2_SERVICE_ENDPOINT, 'GetRatingChanges', {
+            'StartDate': start_date.strftime('%Y-%m-%d'),
+            'EndDate': end_date.strftime('%Y-%m-%d'),
+            'MaxRows': 500,  # Limit.
+            'StartRowIndex': 0  # Offset.
+        })
+        return requests.get(url).json()
+
+
+def publish_cert(cert_id):
+    _update_certs(cert_id, 'Publish')
+
+
+def remove_cert(cert_id):
+    _update_certs(cert_id, 'RemoveProduct')
+
+
+def search_certs(uuid):
+    """SearchCerts for complete data on a single certificate."""
+    data = None
+    if settings.IARC_V2_MOCK:
+        data = json.loads(file('./mock/SearchCerts.json').read())
+    else:
+        url = urljoin(settings.IARC_V2_SERVICE_ENDPOINT, 'SearchCerts')
+        data = requests.get(url, {'CertID': uuid}).json()
+    return IARCV2RatingListSerializer(data=data).data
```

Comment:
So, I still don't know: should we call the serializer `.save()` method in the iarc_v2 client or build another layer to do that.

If going with the fist option, that'd mean:
- Passing app objects and fetching the cert_id from the database instead of passing the cert_id
- In `search_certs`, passing both and if the response is successful, call `save()` *and* associate the cert_id to the app in the database as well.

If going with the second option, that'd mean:
- Leaving the serializer outside of this thin client

IMHO the whole thing is simple enough that we don't need an extra layer. But it implies some more refactoring here.